src/Files.App/ServicesImplementation/Settings/PreferencesSettingsService.cs

```diff
@@ -63,6 +63,12 @@ public List<string> LastSessionTabList
 			set => Set(value);
 		}
 
+		public List<string> LastCrashedTabList
```

Comment:
How is this different from `LastSessionTabList`?

Reply:
`LastSessionTabList` is updated to "Home" tab at startup, presumably in case of a crash without notification. So I added another one that is not updated except during crashes.